browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1269,6 +1269,12 @@ Are you sure you want to do this?
       <message name="IDS_SYNC_CAMERA_PERMISSION_WAS_NOT_GRANTED_TOAST" desc="Popup message displayed when user did not granted camera permission when joining the sync chain by scanning QR code">
         Camera permission was not granted
       </message>
+      <message name="IDS_PASSWORD_SYNC_TYPE_SET_SCREEN_LOCK" desc="Text prompting user to set up screen lock on their device in order to change Sync passwords type">
+        To enable Passwords sync type, first set a screen lock on your device
+      </message>
+      <message name="IDS_ENABLING_PASSWORD_SYNC_AUTH_MESSAGE" desc="Description shown when we ask for auth while enabling passwords sync">
+        Unlock to enable passwords Sync
```

Comment:
I am not sure we need capitalization here. I would suggest:
* To enable passwords sync, first set a screen lock on your device
* Unlock to enable passwords sync

Reply:
After your comment the initial strings seem strange to me, so I applied your suggestions at https://github.com/brave/brave-core/pull/19477/commits/22e17b6f0cef598132748026db2699bda7acf81b 
, because it looks better. Thanks.